### PR TITLE
Feat  my skills

### PR DESCRIPTION
**Pull request by austinmargarone**
@brandonetter 
I can't seem to get the my skills text to center. I think the breakpoints are looking better but on smaller devices the components look to be off to the left. Not sure what i'm doing wrong, probably not the right breakpoints. How do I figure out which breakpoints to use and how to center on smaller devices?

### CODE REVIEW

File: app/globals.css [hunk 1]
```diff
@@ -104,117 +222,56 @@
...
-    font-size: 1.125rem;
-    font-style: normal;
-    font-weight: 400;
-    line-height: 160%;
+  .skillslg {
+    display: flex;
+    width: 90rem;
+    padding: 4.5rem 5.3125rem;
+    flex-direction: column;
+    align-items: flex-start;
```
Comment (brandonetter) [inline app/globals.css]:
```suggestion
    align-items: center;
```
This is why the text isn't centered. Flex-start will place it at the start of the flexbox.

**Review by brandonetter — CHANGES_REQUESTED** [review]
To answer "how do I determine what breakpoints to use to center text on smaller devices", you don't. With tailwind your smallest breakpoint is the default. Remember to think of it as "mobile first".

Your breakpoints change things as your window gets **larger** not **smaller**

**Review by austinmargarone — COMMENTED** [review]
Gotcha. It won't let me resolve conflicts on this branch.

File: app/globals.css (hunk 1 above)
Comment (austinmargarone) [reply app/globals.css]: That'll do it!

**Review by austinmargarone — COMMENTED** [review]
Reviewed

**Comment by brandonetter** [comment]
👍

**Pull request merged**